README.md

```diff
@@ -37,46 +37,46 @@ Please note: Newer Keptn versions might be compatible, but compatibility has not
+Alternatively it is also possible to install the *job-executor-service* in the [same namespace as Keptn](#installation-in-the-same-namespace-as-keptn). 
 
-In order to do the latter, job-executor-service helm chart provides some values that can be configured:
-
-* `remoteControlPlane.enabled` - needs to be set to `true` in order to connect to the remote Keptn instance
+In order to install the *job-executor-service* on the remote execution plane, some values of the helm chart can be configured:
 * `remoteControlPlane.topicSubscription` - list of Keptn CloudEvent types that this instance should listen to, e.g., `sh.keptn.event.remote-task.triggered`
 * `remoteControlPlane.api.protocol` - protocol (`http` or `https`) used to connect to the remote control plane
-* `remoteControlPlane.api.hostname` - Keptn API Hostname (e.g., `1.2.3.4.nip.io`)
+* `remoteControlPlane.api.hostname` - Keptn API Hostname (e.g., `1.2.3.4.nip.io` / `api-gateway-nginx.keptn`)
 * `remoteControlPlane.api.token` - Keptn API Token (can be obtained from Bridge)
 
 **Example**
 ```bash
 KEPTN_API_PROTOCOL=http # or https
 KEPTN_API_HOST=<INSERT-YOUR-HOSTNAME-HERE> # e.g., 1.2.3.4.nip.io
- KEPTN_API_TOKEN=<INSERT-YOUR-KEPTN-API-TOKEN-HERE>
+KEPTN_API_TOKEN=<INSERT-YOUR-KEPTN-API-TOKEN-HERE>
 
 TASK_SUBSCRIPTION=sh.keptn.event.remote-task.triggered
 
 helm upgrade --install --create-namespace -n <NAMESPACE> \
   job-executor-service https://github.com/keptn-contrib/job-executor-service/releases/download/<VERSION>/job-executor-service-<VERSION>.tgz \
- --set remoteControlPlane.enabled=true,remoteControlPlane.topicSubscription=${TASK_SUBSCRIPTION},remoteControlPlane.api.protocol=${KEPTN_API_PROTOCOL},remoteControlPlane.api.hostname=${KEPTN_API_HOST},remoteControlPlane.api.token=${KEPTN_API_TOKEN}
-
-
+ --set remoteControlPlane.topicSubscription=${TASK_SUBSCRIPTION},remoteControlPlane.api.protocol=${KEPTN_API_PROTOCOL},remoteControlPlane.api.hostname=${KEPTN_API_HOST},remoteControlPlane.api.token=${KEPTN_API_TOKEN}
 ```
 
+Please replace `<VERSION>` with the actual version you want to install from the compatibility matrix above or the
+[GitHub releases page](https://github.com/keptn-contrib/job-executor-service/releases).
+
 To verify that everything works you can visit Bridge, select a project, go to Uniform, and verify that `job-executor-service`  is registered as "remote execution plane" with the correct version and event type.
 
+### Installation in the same namespace as Keptn
+
+To install the *job-executor-service* in the same namespace as Keptn you have to disable the `remoteControlPlane.enabled` value as shown below.
+But be aware that all jobs will be executed in this namespace.
+
+```bash
+helm upgrade --install --create-namespace -n <NAMESPACE> \
+  job-executor-service https://github.com/keptn-contrib/job-executor-service/releases/download/<VERSION>/job-executor-service-<VERSION>.tgz \
+ --set remoteControlPlane.enabled=false
+```
```

Comment:
I think we should remove this option. Installing it on the same namespace should not be possible, especially in the context of Kubernetes hardening, separation makes sense.